taker-frontend/package.json

```diff
@@ -36,6 +37,10 @@
     "vite": "^2.6.14",
     "web-vitals": "^2.1.2"
   },
+  "dependenciesComments": {
```

Comment:
wow, this is an amazing feature for documenting such messy issues in 3rd-party dependencies! does cargo have something similar for Rust?

Reply:
It is not a feature as such, just the only way of adding comments to JSON :rofl: